src/researchhub_document/views/researchhub_unified_document_views.py

```diff
@@ -293,7 +293,7 @@ def _get_featured_documents_queryset(self):
     def get_filtered_queryset(self):
         qs = self.get_queryset().filter(is_removed=False)
         qs = self.filter_queryset(qs)
-        return qs
+        return qs.prefetch_related("concepts")
```

Comment:
Add the prefetching to `researchhub_document/filters.py` under the `document_type_filter` method

Reply:
done